bls/signature.go

```diff
@@ -8,23 +8,25 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"errors"
+	"fmt"
 	"strings"
 
 	flowCrypto "github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/crypto/hash"
 )
 
-// The SigningAlgorithm for BLS keys is BLSBLS12381 which is BLS on the BLS 12-381 curve.
+// The signingAlgorithm for BLS keys is BLSBLS12381 which is BLS on the BLS 12-381 curve.
 // This is the only supported BLS signing algorithm in the flowCrypto package.
 // BLS is used such that we can aggregate signatures into one signature.
-const SigningAlgorithm = flowCrypto.BLSBLS12381
+const signingAlgorithm = flowCrypto.BLSBLS12381
```

Comment:
This `signingAlgorithm` const and `hashingAlgorithm` variable are implementation details of how this package wraps the flow BLS scheme implementation underneath.

We don't want this package to expose them externally, so it's best to make them private.